x-pack/plugins/security_solution/public/hosts/components/host_score_over_time/index.tsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useCallback } from 'react';
+import {
+  Chart,
+  LineSeries,
+  ScaleType,
+  Settings,
+  Axis,
+  Position,
+  AnnotationDomainType,
+  LineAnnotation,
+  TooltipValue,
+} from '@elastic/charts';
+import { euiThemeVars } from '@kbn/ui-shared-deps-src/theme';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingChart, EuiText, EuiPanel } from '@elastic/eui';
+import styled from 'styled-components';
+import { chartDefaultSettings, useTheme } from '../../../common/components/charts/common';
+import { useTimeZone } from '../../../common/lib/kibana';
+import { histogramDateTimeFormatter } from '../../../common/components/utils';
+import { HeaderSection } from '../../../common/components/header_section';
+import { InspectButton, InspectButtonContainer } from '../../../common/components/inspect';
+import * as i18n from './translations';
+import { useHostsRiskScore } from '../../../common/containers/hosts_risk/use_hosts_risk_score';
+import { PreferenceFormattedDate } from '../../../common/components/formatted_date';
+
+export interface HostRiskScoreOverTimeProps {
+  hostName: string;
+  from: string;
+  to: string;
+}
+
+const RISKY_TRESHOULD = 70;
+const DEFAULT_CHART_HEIGH = 250;
+const QUERY_ID = 'HostRiskScoreOverTimeQuery';
+
+const StyledEuiText = styled(EuiText)`
+  font-size: 9px;
+  font-weight: ${({ theme }) => theme.eui.euiFontWeightSemiBold};
+  margin-right: ${({ theme }) => theme.eui.paddingSizes.xs};
+`;
+
+const LoadingChart = styled(EuiLoadingChart)`
+  display: block;
+  text-align: center;
+`;
+
+const HostRiskScoreOverTimeComponent: React.FC<HostRiskScoreOverTimeProps> = ({
+  hostName,
+  from,
+  to,
+}) => {
+  const timeZone = useTimeZone();
+
+  const dataTimeFormatter = useMemo(() => histogramDateTimeFormatter([from, to]), [from, to]);
```

Comment:
nit: i dont like that a `useMemo` and `useCallback` are named the same `Formatter`, could you reconsider these names

Reply:
They are both passed down to the same property `tickFormat`.  What do you think about naming it `memoizedDataTimeFormatter`?